tests/test_integration_workflows.py

```diff
@@ -257,6 +259,15 @@ def run_inference_test(root_dir, model_file, device="cuda:0", amp=False, num_wor
         CheckpointLoader(load_path=f"{model_file}", load_dict={"net": net}),
     ]
 
+    saver = SaveImage(output_dir=root_dir, output_postfix="seg_handler")
+
+    def save_func(engine):
```

Comment:
Hi @wyli ,

I think we can just add the `SaveImaged` transform to the postprocessing for this case, for example:
https://github.com/Project-MONAI/tutorials/blob/main/modules/engines/unet_evaluation_dict.py#L95

Thanks.

Reply:
thanks, that already exists in this test: https://github.com/Project-MONAI/MONAI/blob/256dbab8532bf67a2f8d3d1b0174829c69bc9858/tests/test_integration_workflows.py#L252-L254

Reply:
OK, then may I how this PR fixed the integration test issue?
I see it just added a new saving logic?

Thanks.

Reply:
yes it tests two types of savings... https://github.com/Project-MONAI/MONAI/blob/256dbab8532bf67a2f8d3d1b0174829c69bc9858/tests/test_integration_workflows.py#L342-L343